### ISSUE TITLE
[Discovery] Logged-in homepage: Final synthesis

### ISSUE THREAD

**Issue by Samara-Strauss** [department-of-veterans-affairs/va.gov-team#8839]
## Background

We need to put together a final discovery synthesis that summarizes the most important findings from the stakeholder interviews, user research, comparative analysis, and analytics review. We also need to make any final recommendations we have about how to approach this project.

## Tasks

- [x] Determine high level recommendations.
- [x] Create outline for report.
- [ ] [Write and iterate on report](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/logged-in-homepage/2.0-redesign/discovery-and-research/discovery-summary.md).

## Acceptance criteria

- [ ] Final discovery synthesis is complete.

**Comment by Samara-Strauss** [comment]
@andaleliz [I added my outline here](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/logged-in-homepage/2.0-redesign/discovery-and-research/discovery-summary-outline.md) and also moved the initial discovery summary thoughts we had to that page.

**Comment by Samara-Strauss** [comment]
Hey @andaleliz! I am SO sorry this took all day, but I finally had a chance to update the [key findings section of the outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/logged-in-homepage/2.0-redesign/discovery-and-research/discovery-summary-outline.md#key-findings). Let me know what you think.

- I kept the main takeaway as people want a task-based, personalized homepage. This is the validation we needed to feel justified in continuing with this work, and I think it's a good way to lead this section.
- I then divided the other sections under headings — content, design, and overall authenticated experience. I don't know if those groupings are good or if they matter, but it felt like a way to bring some extra structure to this section. Please let me know if you think they're a bad idea, break flow, or generally aren't helpful.
- From your doc, I moved over:
  - Mentioning in the first key finding that veterans want a task-based homepage in addition to a personalized one.
  - Talking about how important records and documents are.
  - Mention of the blue sky prototype and that, generally speaking, we could learn from it.
- I also moved the discussion about developing design patterns for authenticated tools down to the process section.

Otherwise, I think the rest of what was in your doc was either already represented in the outline or might be too granular (but perfect for a detailed, informal doc that we hand off to Tressa). Let me know your thoughts!

**Comment by Samara-Strauss** [comment]
Btw, I did not have time to think about the Projects section under the Recommendations heading today, so that is largely unchanged. I'll continue to work on that. Otherwise, I think the rest of the outline is all set.

**Comment by andaleliz** [comment]
Hey @Samara-Strauss, no worries! Totally understand when other things come up.

I can relate to wanting to add section headings for structure - I did the same thing for the user research synthesis! I ended up taking them out during my 3rd or 4th revision, once things felt more organized and clear overall. I think they're helpful as we refine the report, and I vote to leave them in for now. We can always remove once things are more solidified, if we think they'll feel like noise to our audience.

In terms of what you moved over, I think all the bases are covered now! Thanks for doing that.  

I added the recommendation about younger Veterans to the process section, under `user test, user test, user test` because I really don't want that to get lost.

I'll work on adding details to the key findings. There are a couple that I think need to be reworded to reflect what we saw across multiple parts of this discovery (rather than just user sessions), but I'll point those out in a separate comment once I've updated the document.

Oh, and RE recommendations - posted in Slack about this and Shawna's suggestion.  I like the idea of involving the team somehow with these before we consider it final.

**Comment by andaleliz** [comment]
Also, I think a Discovery Goals section would be helpful (between background and activities) to give more context to why we chose those specific discovery activities, especially for someone who is stepping into this work for the first time.

**Comment by andaleliz** [comment]
@Samara-Strauss I added details to [the first three findings](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/logged-in-homepage/2.0-redesign/discovery-and-research/discovery-summary-outline.md#people-want-a-task-based-logged-in-homepage-tailored-to-them).

I'd love you feedback on a couple of points before I go further:
- how does this line up with the level of detail you have in mind for this report?
- I'm not sure how much evidence we should provide from each activity to make a strong case for the finding; so far have been trying to take it 1 step further than just "in this activity we saw xyz" and back it up with a link or specific detail.

And of course, anything else you'd like to share feedback on. Thanks in advance!  :)

**Comment by Samara-Strauss** [comment]
> I added the recommendation about younger Veterans to the process section, under user test, user test, user test because I really don't want that to get lost.

Yes, thank you! Sorry I missed that.

> I'll work on adding details to the key findings. There are a couple that I think need to be reworded to reflect what we saw across multiple parts of this discovery (rather than just user sessions), but I'll point those out in a separate comment once I've updated the document.

Yes, please feel free to reword. The wording is NOT final and not sacred. Most of it was done very quickly so another eye and further refining is much appreciated :)

> Oh, and RE recommendations - posted in Slack about this and Shawna's suggestion. I like the idea of involving the team somehow with these before we consider it final.

Agreed! I am going to have a plan to share with the team so we can get feedback and update before sharing to the larger VA team.

> Also, I think a Discovery Goals section would be helpful (between background and activities) to give more context to why we chose those specific discovery activities, especially for someone who is stepping into this work for the first time. I brought over bullets from the original research needs doc you created at the beginning of this work.

Great idea. I would say let's keep this as brief as we can so we can get people to the good stuff quickly, but yes, I agree this makes sense.

**Comment by Samara-Strauss** [comment]
> @Samara-Strauss I added details to the first three findings.
> 
> I'd love your feedback on a couple of points before I go further:
> 
> - how does this line up with the level of detail you have in mind for this report?
> -  I'm not sure how much evidence we should provide from each activity to make a strong case for the finding; so far have been trying to take it 1 step further than just "in this activity we saw xyz" and back it up with a link or specific detail.
> - anything else you'd like to share feedback on.

- Overall, things are definitely going in the right direction! It is SO EXCITING to see this come together already. 
- Re: how much evidence to present — We should pick the strongest points to make the case clear from the report, whether that is analytics or an explanation of what happened from one of our other activities. And cite everything! Always link to the original report.
- Also, we don't really need more than a paragraph (not including quotes or implications) to explain most of these. You've already kept things within that limit, so that looks good to me.
- One thing though on explanation — be as specific as you can. If we get too generic, the report starts to lose meaning. An example of this is "and this also aligns with what we saw across sites in our comparative analysis." While you link to the report (which is great), this doesn't really tell anyone what we saw in the comparative analysis (dashboards that emphasized personal information over generic recommendations or content, which is the private sector standard). Just make sure that it's clear what we saw and where so that the reader can (if they want) read into the full reports but they don't have to.
- Btw, I know it was my idea but let's take out the additional Content/Design/Authenticated experience overall headings. They cause us to lose the actual findings among the explanations since the actual findings are now h4s instead of h3s.

Some specific feedback below:

**People want a task-based logged-in homepage tailored to them.**

- I don't think we need to say the thing about 10/10 participants — we can just say all participants created a task-based homepage and knew exactly what they needed to manage their experience with the VA. 
- LOVE the bit about links being clutter and noise if they're not relevant to the user. Do we have quote for this?
- I think we can add one sentence about analytics too and link to the report — we can see that logged-in users are very task oriented compared to logged-out users.
- "Stakeholders described this type of task-based, personalized experience as one of their goals during our interviews, and this also aligns with what we saw across sites in our comparative analysis." Let's answer what we saw in the comparative analysis just to make this a bit more specific. "Stakeholders described this type of task-based, personalized experience as one of their goals during our interviews. We also saw this standard set in our comparative analysis, in which every private sector site we looked at anchored their logged-in experiences in a dashboard and navigation that was different from their logged-out marketing sites."

**Health care and disability are king.**

- Can you please link to the analytics findings in the analytics report?

**Comment by andaleliz** [comment]
This is great feedback, thanks @Samara-Strauss! Going to run with this.  

RE the `noise and clutter`  - the participant who gave the the horn tooting quote was referring to all the extraneous content as the VA tooting their horn.   I'll go back through and see if I can find any other more specific quotes about that though :)

Toot toot!
(really want to make sure this comment goes down has having the most toots in one issue)

**Comment by andaleliz** [comment]
@Samara-Strauss I've updated the [final findings summary doc](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/logged-in-homepage/2.0-redesign/discovery-and-research/discovery-summary.md) w/ the first few sections - let me know what you think! 

**Question:** 
Would it be okay for me to add the HKS PDF file to our repo somewhere, and then link to it in our doc? I think it's worth noting, but not sure if there is sensitivity around sharing that out.  

**Note:** 
You had a finding written as "Not everything needs to be a “widget” — the best structure highlights the most important information and is easy to scan."

I updated to what you see in the doc (focused on widgets vs links) because the widget vs structure part feel a little disjointed to me combined. Structure and scan-ability don't always go hand in hand with widgets, and how people get things done.  It also hints at someone saying everything DOES have to be a widget, which isn't the case afaik.

Finally, I initially split this into two findings, with one addressing the structure. As I wrote it out, it felt redundant to the comments about the blue sky prototype and notifications, which I think are more important and could concretely be tied to multiple activities in our overall discovery effort.

Totally open to discussion and revising if you disagree and want me to work on detailing out something on structure, or bring them back together.

**Comment by Samara-Strauss** [comment]
Hey @andaleliz!! SO excited to check out the first few sections of the final summary! I'm going to do that tomorrow but wanted to acknowledge receipt before leaving for the day.

I also wanted to let you know I more fully filled out the details for how I see us carrying out the logged-in homepage project under [the Recommendations heading of the outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/logged-in-homepage/2.0-redesign/discovery-and-research/discovery-summary-outline.md#recommendations). I would love your feedback on that section. I haven't made any additional changes to the Process or Next Steps sections, but I think those are OK for me to start working on, and I'll get those to you for feedback when they actually have more fully fleshed out content in them.

> Would it be okay for me to add the HKS PDF file to our repo somewhere, and then link to it in our doc? I think it's worth noting, but not sure if there is sensitivity around sharing that out.

Yes! This is actually [already in our repo but in a different team folder](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/va-mobile-app/HKS-research/Harvard-VA%20Executive%20Presentation%20.pptx). No sensitivity to sharing that out.

> You had a finding written as "Not everything needs to be a “widget” — the best structure highlights the most important information and is easy to scan."
> 
> I updated to what you see in the doc (focused on widgets vs links) because the widget vs structure part feel a little disjointed to me combined. Structure and scan-ability don't always go hand in hand with widgets, and how people get things done. It also hints at someone saying everything DOES have to be a widget, which isn't the case afaik.
> 
> Finally, I initially split this into two findings, with one addressing the structure. As I wrote it out, it felt redundant to the comments about the blue sky prototype and notifications, which I think are more important and could concretely be tied to multiple activities in our overall discovery effort.
> 
> Totally open to discussion and revising if you disagree and want me to work on detailing out something on structure, or bring them back together.

I'll give everything a full review tomorrow, but this all sounds good to me. You make a good point that my initial wording in the outline re: widgets wasn't really communicating the point we were trying to get across, so your updates sounds like they provide more clarity. I'll let you know if I have any specific feedback after I'm able to give this a read.

**Comment by andaleliz** [comment]
@Samara-Strauss the recommendations section looks good -definitely reasonable and actionable from the findings.  These recommendations serve as a plan for moving forward, at least to me, so there are just a few things I would consider updating when you start writing it out in more detail:

- Add some details about when/how MyVA will be retired.  (part of an existing phase? A separate phase on its own?). 
>Phase 3: Focus on benefits discovery
Some of this will inevitably be part of phase 1 since we can’t launch without a “blank” state.
- I think Benefits discovery **will** have to be part of phase one, so it would be good to add some specifics about what could be in scope for that phase. I think it'd rely on what is possible to display from Top Tasks based on what we know about the logged in users...maybe there are some "getting started with the VA" docs or content we could elevate for those users where the widgets will live for established Veterans.

**Questions**
>Out of scope:
Moving over "Find VA Benefits" functionality or content

Would this be in scope for phase 3?

There aren't any specific recommendations about improving access to documents/records. What are your thoughts around that? 

Looping in @sshein for awareness - sorry to ping you so late in the thread on this issue! I just realized I hadn't done it sooner.

**Comment by Samara-Strauss** [comment]
Hey Liz! Here are my notes on the report so far:

### Background

- When you link to “Personalization research conducted in 2019”, can you link to the full Personalization 2.0 strategy? That report gives the full context of all the research we did and the final recs based on that research.
- In the first sentence or two, let’s mention that we conducted this discovery effort in order to guide the logged-in homepage redesign. 
- I also don’t think we need to go into as much detail about the background of the old Vets.gov homepage and how it became My VA. 
- Maybe start with something like “Based on the 2019 Personalization 2.0 strategy, the authenticated experience team is pursuing a redesign of the existing logged-in homepage on VA.gov. This will involve retiring the current My VA dashboard and bringing people’s personal information right to their homepage.” And then we can go into more detail on needing a discovery effort to dive in deeper for the logged-in homepage redesign and what we did as part of discovery.

### Discovery Goals

- Can you capitalize “VA” in all instances of “VA.gov”?

### Discovery activities

- IMO, I think we should combine these with the background section so that we are answering the questions “What is this, and what did you do” in the same space.

### People want a task-based logged-in homepage tailored to them.

- I don’t think we need to set a ton of background here, but I would mention that this conclusion is based on user research. Otherwise, the reader is left wondering how we came to this conclusion.  “We conducted user research in which we asked participants to create the page they wanted to see when they logged in. One thing was clear in these sessions — veterans want a task-based, personalized homepage. All participants…”

### Health care and disability are king.

- I think this would be stronger if we opened with a single sentence that clarifies this finding before going into the supporting evidence. “Veterans are primarily using the VA's digital tools to manage their health care and the disability claims process. In user research, we saw…”
- “We should provide users with direct ways to complete health and disability tasks.” What does "direct ways" mean? We should either clarify or remove. Implication 1 is strong on its own if we decide to remove.

### Veterans care about anything they have in flight, whether that is education or other benefits.

- For the implications, the first two don’t really follow the format of the rest of the implications so far, which is essentially a general recommendation.  For this section, I think implication 3 is the strongest one in the list and can probably stand on its own.

### Records and documents are a focus for Veterans.

- “We should create more direct access to records and documents…” Since we already have direct access to records from the homepage, I would change the beginning of this sentence to something like “We should keep the direct access to records and documents that we already have on the homepage…”

### The logged-in homepage needs to scale to support every phase of the Veteran lifecycle.

- Great tie in with the veteran journey maps!!
- “This tells us we need to account for more than tasks on the logged-in homepage.” What does this mean? This needs clarification. I think you mean that we need to account for exploring and learning about new benefits.
- Under implications, “We need to account for more than just tasks on the logged-in homepage”, needs similar clarification.
- Implications 1 and 3 essentially say the same thing, so I would consolidate.

### The existing "blue sky" prototype gets a lot of things right.

- Would love a screenshot of the blue sky prototype here.
- “Using this pattern as a starting point would save design time and allow us to ramp up more quickly than if we start from scratch.” What do you mean by this? It sounds like you are saying we should just copy the design and implement as-is, but obviously this wouldn’t be possible.

### Veterans expect a mix of links and widgets to complete tasks from their homepage.

- OK, so this is on me, but I feel like maybe we’re using the term “widget” in a way we shouldn’t be. What I want to emphasize is that there needs to be a balance between personalized information  (content blocks) we show right on the dashboard and linking to tools so the page doesn't become too cluttered. In truth, I don’t think we’ll have true widgets where you can complete tasks on the homepage in the MVP. Maybe in a future version, but not sure. What do you think of that? Do you also feel we need to de-emphasize the term “widget” in favor of something else?
- “straightforward tasks like refilling a prescription or scheduling an appointment” At the VA, I don’t know if these are actually straightforward and could ever be completed from the homepage.
- The implications aren’t clear or actionable to me. IMO, the implication for this section is that we should only feature content for the most important tasks/benefits on this page, and link to other tasks and information. Not everything should have a content block.

### The homepage needs to be supported by a cohesive logged-in navigation.

- For implications, I would combine the first two implications into one and make it something actionable. “We should provide a logged-in navigation that is task focused to help users get to tasks more quickly from anywhere on the site.”

### Veterans want new information to be clear to them when they log in, and expect to receive new information through multiple channels.

- I think we need to be more transparent here and mention that the main way people want to receive notifications is through email and text. People did want any updates to also be available to them when they log in, but this was secondary to receiving notifications outside of the site.
- The implications need work:
  - Are we recommending a separate Notification Center?
  - We should clarify what in-line notifications means.
  - Veterans are used to paper mail and calls from the VA, not emails. They are used to emails and texts from the private sector and seem to want that from the VA, too.

### General

- I didn’t edit for grammar but noticed some minor issues. I am assuming we’re just working on getting the content right and not to worry about some of the finer details. Is that an accurate assumption?
- I would love if we could fit a screenshot of one of the user dashboards created during our research sessions into this doc for some visual interest, but not quite sure where. Let me know if you think that’s appropriate.
- I would use the quote style or bold for veteran quotes so they stick out more.

**Comment by Samara-Strauss** [comment]
> Add some details about when/how MyVA will be retired. (part of an existing phase? A separate phase on its own?).

Awesome idea. This would be part of phase 1 with the launch of the new logged-in homepage. Will include this info for sure.

> I think Benefits discovery will have to be part of phase one, so it would be good to add some specifics about what could be in scope for that phase. 

Also a great idea. This, at minimum,  needs to account of a user who has no benefits and no applications in flight, essentially a blank state. Otherwise, other discovery/improvements would take place later on.

> Out of scope:
> Moving over "Find VA Benefits" functionality or content
> Would this be in scope for phase 3?

Yeah, I think revisiting this — whether it is a redesign of the feature or a more comprehensive first-time user experience — would be part of phase 3. We'd also likely look more closely at recommending things to people based on what we know about them, eg. "Since you have disability benefits, have you considered voc rehab/whether you need housing assistance/etc"

> There aren't any specific recommendations about improving access to documents/records. What are your thoughts around that?

I actually think we have good access to this on the site already. No specific thoughts right now on how we'd improve this beyond making sure it's featured on the new logged-in homepage (and hopefully someday having access to people's DD214's online).

-----

My goal is to start writing my sections tomorrow barring unforeseen things coming up. I'll send you draft 1 when I have it :D

**Comment by andaleliz** [comment]
Hey @Samara-Strauss ! Thanks for all the feedback. I've made most of the changes you suggested, and have a comments on a few items.  Generally, one thing I should've clarified is that I see implications as walking a fine line between conclusions and recommendations.  Although many of them do end up as actionable recommendation, I had included points I thought were interesting, and implied by the findings, even if I know they are not possible at this time, or actionable. It was my attempt to be thorough and true to the data.  I still stayed true to the data, but updated the majority to be actionable so they're more useful to our audience.

As always, open to discussing any of this!

**Specific comments:**

>“Using this pattern as a starting point would save design time and allow us to ramp up more quickly than if we start from scratch.” What do you mean by this? It sounds like you are saying we should just copy the design and implement as-is, but obviously this wouldn’t be possible.

I wasn't suggesting a literal copy-and-paste, but definitely saying using something along these Iines would speed things up.  Knowing that we want a phased approach and can't jump into something like this, I've updated that implication to read: **"Using this pattern could save design time as we roll out a more mature logged-in homepage."**

> OK, so this is on me, but I feel like maybe we’re using the term “widget” in a way we shouldn’t be. What I want to emphasize is that there needs to be a balance between personalized information (content blocks) we show right on the dashboard and linking to tools so the page doesn't become too cluttered. In truth, I don’t think we’ll have true widgets where you can complete tasks on the homepage in the MVP. Maybe in a future version, but not sure. What do you think of that? Do you also feel we need to de-emphasize the term “widget” in favor of something else?

The question of widgets vs links came from our research questions in the user study and comparative analysis, and the whole time I approached getting the answer to this through a lens of a widget being something that helps you complete a task, not something that just shows personalized information. Given that understanding, I think widget is appropriate, but I fear that I misunderstood the goal of this research question and perhaps the data we got doesn't align with what you were hoping to find out.

> “straightforward tasks like refilling a prescription or scheduling an appointment” At the VA, I don’t know if these are actually straightforward and could ever be completed from the homepage.

I updated this to say "tasks Veterans perceive to be straightforward".  They really did talk about these things as if they should be easy and simple.

> The implications aren’t clear or actionable to me. IMO, the implication for this section is that we should only feature content for the most important tasks/benefits on this page, and link to other tasks and information. Not everything should have a content block.

I updated the implications I had to be more succinct and actionable, but am not getting quite the same implication you are from these particular findings.  I feel only featuring content for the most important tasks and benefits is implied in the first, and maybe second, findings, whereas this one addresses more how users would go about actually doing it.  I'm totally open to adding what you wrote to the implications if you still disagree.

> Veterans want new information to be clear to them when they log in, and expect to receive new information through multiple channels.

> I think we need to be more transparent here and mention that the main way people want to receive notifications is through email and text. People did want any updates to also be available to them when they log in, but this was secondary to receiving notifications outside of the site.

I updated this section to more clearly tell the story of what we saw in our user sessions. I didn't get that people necessarily *want* emails, it was more that it was their expectation based on previous/current experience with the VA.  I do remember one participant saying texts were helpful but I don't recall texts coming up much in the sessions, and another said they preferred email to phone messages.  Possible that I'm misremembering the frequency that texts came up though. Let me know if you want me to revisit the notes. 

> Are we recommending a separate Notification Center?

This is where the fine line is between implications and recommendations. **Based on the research**, I can't confidently recommend a separate notification center for this work. Participants didn't explicitly create one or go into details around what they'd expect there.  Most of the people who sent me screenshots for our comparative analysis didn't have a lot of notifications so I wasn't able to evaluate that thoroughly.  That being said, I definitely recommend it from a modern-UX perspective, and think Veterans would find it useful once they know it's there, but I don't think the findings imply that specifically.

> Veterans are used to paper mail and calls from the VA, not emails. They are used to emails and texts from the private sector and seem to want that from the VA, too.

Multiple participants referred to getting emails about appointments and messages from their doctor, which is why I mentioned that in the finding. Again, I didn't get that it is what the necessarily *want* but it is more of an expectation.

> I didn’t edit for grammar but noticed some minor issues. I am assuming we’re just working on getting the content right and not to worry about some of the finer details. Is that an accurate assumption?

Should all be good now. I did have this proofread ahead of time, but we must've missed some issues. I ran all of this through grammarly as well as triple checking so please let me know if you find anything else.

>  I would love if we could fit a screenshot of one of the user dashboards created during our research sessions into this doc for some visual interest, but not quite sure where. Let me know if you think that’s appropriate.

Love this idea, and also think it's a good way to clarify in-line notifications. See what you think about the screenshots I picked.

**Comment by Samara-Strauss** [comment]
Hey @andaleliz! [Here is my first pass at my sections](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/logged-in-homepage/2.0-redesign/discovery-and-research/discovery-summary.md#moving-forward-project-approach). Let me know what you think!

I'll give your updates another read through hopefully tomorrow, but want to respond to your comments above first:

> I've updated that implication to read: "Using this pattern could save design time as we roll out a more mature logged-in homepage."

Can we update to something like "Modeling after these patterns could save design time..." to make this a little clearer?

> I approached getting the answer to this through a lens of a widget being something that helps you complete a task, not something that just shows personalized information. Given that understanding, I think widget is appropriate, but I fear that I misunderstood the goal of this research question and perhaps the data we got doesn't align with what you were hoping to find out.

I think you understood things just fine. To be clear, anything we show on the homepage will, at the very least, have a link to a more full flow/info. So, in that sense, everything will be actionable. But with regards to full task completion (eg. refilling a prescription) on the homepage itself, I am genuinely unsure when or if this will be possible. I just want to make sure we don't promise something we can't deliver on. 

> I updated this to say "tasks Veterans perceive to be straightforward".

Sounds good!

> I updated the implications I had to be more succinct and actionable, but am not getting quite the same implication you are from these particular findings. 

Let me go back and read through them again with your comments above in mind.

> I updated this section to more clearly tell the story of what we saw in our user sessions. I didn't get that people necessarily want emails, it was more that it was their expectation based on previous/current experience with the VA. I do remember one participant saying texts were helpful but I don't recall texts coming up much in the sessions, and another said they preferred email to phone messages. Possible that I'm misremembering the frequency that texts came up though. Let me know if you want me to revisit the notes.

Hey, I'm sorry, I don't think I was clear with my original comment. I think I was worried we were painting on-site notifications as more important than they are and not being clear that this is secondary to meeting veterans where they are through email and text. I'll review your updates. It's probably fine, and sorry for my lack of clarity.

> Based on the research, I can't confidently recommend a separate notification center for this work.

OK, thank you for letting me know. I mentioned it in my section of the write-up but I'll reconsider this.

> Multiple participants referred to getting emails about appointments and messages from their doctor, which is why I mentioned that in the finding. Again, I didn't get that it is what the necessarily want but it is more of an expectation.

I'm sorry, you are totally right. I was thinking VA wide but they DO actually get email notifications from their doctors for secure messages and I think maybe prescription stuff through MyHealtheVet.

**Comment by andaleliz** [comment]
@Samara-Strauss 

> I just want to make sure we don't promise something we can't deliver on.

Will our audience feel we're promising something if we put it in the report as a finding, or implication? I can see people might have expectation that we'll act on all recommendations, but I think findings and implications are about more of stating facts and what we can reasonably conclude from those facts, rather than a plan of action.

>OK, thank you for letting me know. I mentioned it in my section of the write-up but I'll reconsider this.

With regard to the notification center - I don't know that you need to reconsider this. Just my opinion, but I feel it's reasonable to include something in the recommendations that is informed by both the research AND our expertise in UX, AE, and personalization. I think notification center is something that **will** be valuable to users, and worth incorporating into a more mature AE. 
 Perhaps the recommendation is that we know this will be useful, but will need to do some more specific research into this area before developing a detailed plan to add this to our experience.

Looking forward to reviewing in close detail tomorrow AM with my morning cup o' coffee :)

**Comment by andaleliz** [comment]
Good morning! You've outlined a really solid approach that feels manageable and reasonable.  It makes me so excited to see where this goes! It doesn't sound like it would take *that* long for the first phase to roll out. So exciting!  

- Excellent idea to specifically say we are **not** changing the logged-out homepage, I'm glad you included that. 
- Thank you for specifically calling out records and documents in phase 1!
- I think the balance of info in the process section is :chefs-kiss: spot on.
- I noticed a couple of very small editorial things:

> Retiring My VA

​		That's the only bullet in the doc w/out a period at the end (from Phase 1, In Scope section)

> Showing new data, or data in a different way (eg. could we should where a prescription is en route?).

​		I *think* you mean could we show instead of should?  



And then a few other thoughts/questions. I'm fully behind this doc as-is, so please just take these as suggestions. 

- The write-up might benefit from a little more elaboration on what will change from the current logged-in homepage so that people get excited and start to imagine where we're going.  We had a discussion where we talked about putting the My VA content above the top 20 links and leaving all the content lower down the page as-is, as part of phase 1... how does that continue to evolve in Phase 2?  The way this reads doesn't leave me feeling that we're going to end up with an experience similar to what we see in the private sector, where the unauthenticated homepage is radically different from the auth homepage.  

- These aren't recommendations as much as they are a plan. I typically like to include recommendations in a  summary of findings for a big research effort like this.  I'm wondering if I should update what I'm calling **Implications** to **Recommendations**, and then ensure each one actually IS an actionable recommendation. I think almost all of them are already.  What are your thoughts? I could totally be over thinking this.

- I'm wondering if we need to be a more specific about how analytics will play into each phase? In the intro for phase 2 you mention it, but wondering if it would be helpful to folks to understand when/how we plan to measure, and to inform which parts of the phase.  Perhaps that comes in a more specific product outline though.

- On that note, under Stakeholder Management, I might update this bullet to include analytics:

  > - Lead with strong recommendations based on user insights, and analytics data, when sharing work.

**Comment by Samara-Strauss** [comment]
> Will our audience feel we're promising something if we put it in the report as a finding, or implication?

Probably not. I might be being overly cautious. I just don't want to leave room for that misinterpretation. I think this probably can be solved by how we word things, so saying we want to "work towards" or "explore" XYZ instead of saying we "should do" XYZ if we're unsure about whether something might ever be possible.

> Just my opinion, but I feel it's reasonable to include something in the recommendations that is informed by both the research AND our expertise in UX, AE, and personalization. 

Great point. I'll plan to leave it in there for now and review to see whether I think additional clarification is needed in my sections.

> Retiring My VA
> ​ That's the only bullet in the doc w/out a period at the end (from Phase 1, In Scope section)
> 
> Showing new data, or data in a different way (eg. could we should where a prescription is en route?).
> ​ I think you mean could we show instead of should?

Thank you for catching these. Fixed!

> And then a few other thoughts/questions. I'm fully behind this doc as-is, so please just take these as suggestions.

There are great points here and I think they'd be easier to discuss on a call. I've reached out on Slack so let me know when you're available.

> Lead with strong recommendations based on user insights, and analytics data, when sharing work.

Added mention of analytics to this bullet point.

Thanks again for all your hard work @andaleliz. This is one of the best things I've worked on here and I am so excited to see this work move forward with the foundation you've worked hard to set :D

**Comment by andaleliz** [comment]
>saying we want to "work towards" or "explore" XYZ instead of saying we "should do" XYZ if we're unsure about whether something might ever be possible.

That makes so much sense. I'll look over what I've written and make updates along these lines as needed.

> Thanks again for all your hard work @andaleliz. This is one of the best things I've worked on here and I am so excited to see this work move forward with the foundation you've worked hard to set :D

Where is the sob emoji?! This is such a nice thing to read, thank you. I'm on the same page - one of the best things I've worked on, anywhere.  I'm glad you're excited and are happy with where we've gone with this discovery.

**Comment by Samara-Strauss** [comment]
@andaleliz here are my latest edits for the readout. There was a lot, which is OK — we're at the point of fine-tuning and this is a long report, so please don't mistake this for me being unhappy with anything. 

My goals in these edits were as follows:

- Tighten up some of the wording to make things more succinct without losing specificity or the spirit of the findings. The report is long, so I want to make sure every word counts. 
- Correct some lingering typos (which are totally inevitable in anything this long) and minor grammar things.

Fortunately, since all edits were made as tracked changes, you can reject any ideas that you think are terrible! So give it a read and feel free to accept what you think makes the report better and reject things if they don't (or if I made something inaccurate by cutting it down too much).

[Logged-in homepage discovery summary_SS edits.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/4732867/Logged-in.homepage.discovery.summary_SS.edits.docx)

**Comment by Samara-Strauss** [comment]
@andaleliz [my section of the readout](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/logged-in-homepage/2.0-redesign/discovery-and-research/discovery-summary.md#moving-forward) is updated based on our convo before. I think it's SO much stronger now and I'm glad we decided to cut out the more detailed stuff that is really for me/the team. Let me know what you think.

**Comment by andaleliz** [comment]
YAS! @Samara-Strauss it IS so much stronger now. I have no edits or feedback other than **this is awesome**.

I also updated my portion. I included all of your edits, and updated each section to include recommendations instead of implications.  I had to rephrase some things but the main points are all still there. 

If you're good with my updates - is anything else we need to do before we can close this out? 

Now that the phases are less pared down, I think that the outcome of the workshop won't really impact the content of this document (rather, it will be useful for the next round of planning).

**Comment by Samara-Strauss** [comment]
@andaleliz thanks! I'm glad the new stuff is stronger 😃 

I also gave things one last read through and made some very minor updates. Otherwise, I think this looks awesome and we can call it done!!

🎉 🎉🎉

**Comment by andaleliz** [comment]
🎉 YAY! So exciting! But, this is definitely a little bittersweet.  All of a sudden it feels like, "...that's everything?"

But, I know it ISN'T, and I can't wait to see where this goes! Also glad we have that workshop next week to wrap it all up.  Closing this issue!
